.changeset/tasty-rockets-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-dropdown": patch
+---
+
+Update dropdown openers to use `role="combobox"` instead of `button`.
```

Comment:
(very) nit: The ActionMenu also uses DropdownOpener and it still uses role="button"!

```suggestion
Update dropdown openers for SingleSelect and MultiSelect to use `role="combobox"` instead of `button`.
```

Reply:
Seems very important to me! Good call!

Reply:
Added another note to the changelog about labeling!